tools/rosgraph/src/rosgraph/xmlrpc.py

```diff
@@ -76,9 +76,32 @@ def isstring(s):
     except NameError:
         return isinstance(s, str)
 
-class SilenceableXMLRPCRequestHandler(SimpleXMLRPCRequestHandler):
 
-    protocol_version = 'HTTP/1.1'
+def check_kernel(major, minor):
+    """Small helper version to check that the current platform kernel is a
```

Comment:
probably it would be nice to keep some description from https://github.com/ros/ros_comm/issues/2118#issue-785600749 for maintenance?